.github/dependabot.yml

```diff
@@ -0,0 +1,28 @@
+# Basic dependabot.yml file
+# https://docs.github.com/en/github/administering-a-repository/configuration-options-for-dependency-updates#package-ecosystem
+
+version: 2
+updates:
+  - package-ecosystem: "npm"
+    directory: "/dashboard"
+    schedule:
+      interval: "daily"
+    open-pull-requests-limit: 2
+
+  - package-ecosystem: "npm"
+    directory: "/integration"
+    schedule:
+      interval: "daily"
+    open-pull-requests-limit: 2
```

Comment:
I would not add here (nor in snyk) the integration package.json, that's only used for testing, it's not part of the application.